Internal patch to only enable the current supported boot source option (Boot
from Image).  This patch will not be committed upstream.

--- horizon-2015.1.2/openstack_dashboard/dashboards/project/instances/workflows/create_instance.py.orig	2015-12-12 12:05:17.628569744 -0500
+++ horizon-2015.1.2/openstack_dashboard/dashboards/project/instances/workflows/create_instance.py	2015-12-12 12:08:12.789789554 -0500
@@ -153,24 +153,7 @@
         source_type_choices = [
             ('', _("Select source")),
             ("image_id", _("Boot from image")),
-            ("instance_snapshot_id", _("Boot from snapshot")),
         ]
-        if base.is_service_enabled(request, 'volume'):
-            source_type_choices.append(("volume_id", _("Boot from volume")))
-
-            try:
-                if api.nova.extension_supported("BlockDeviceMappingV2Boot",
-                                                request):
-                    source_type_choices.append(
-                        ("volume_image_id",
-                         _("Boot from image (creates a new volume)")))
-            except Exception:
-                exceptions.handle(request, _('Unable to retrieve extensions '
-                                             'information.'))
-
-            source_type_choices.append(
-                ("volume_snapshot_id",
-                 _("Boot from volume snapshot (creates a new volume)")))
         self.fields['source_type'].choices = source_type_choices
 
     @memoized.memoized_method
